### DISCUSSION TITLE
Whats the best way to try radiance caching described in the paper

### DISCUSSION THREAD

**Discussion by 3a1b2c3** [NVlabs/instant-ngp#760] [Q&A] [answered]
Whats the best way to try radiance caching described in the paper? is it supported in testbed as well?

**Comment by Tom94** [comment, marked as answer]
Hi there, radiance caching isn't implemented in testbed and there currently is no official source code release.

That said, there are third-party implementations floating around (with varying levels of matching the paper). One that comes to my mind being https://github.com/shocker-0x15/GfxExp.

I'll update this thread when we've publically released neural radiance caching source code. Cheers!

**Reply by 3a1b2c3** [reply]
much appreciated
